### PR TITLE
Add Lambda on Graviton page

### PR DESCRIPTION
**Pull request by julianwood**
This PR adds a Lambda on Graviton page with additional information as well as some simple to deploy demos to explore how to build and migrate functions to Graviton2

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-lambda/README.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+## Migrating x86 Lambda functions to arm64
+### Functions without architecture-specific dependencies or binaries
+Many Lambda functions may only need a configuration change to take advantage of the price/performance of Graviton2. Other functions may require repackaging the Lambda function using Arm-specific dependencies, or rebuilding the function binary or container image.
+
+If your functions don’t use architecture-specific dependencies or binaries, you can switch from one architecture to the other with a single configuration change. Many functions using interpreted languages such as Node.js and Python, or functions compiled to Java bytecode, can switch without any changes. Ensure you check binaries in dependencies, Lambda layers, and Lambda extensions.
+
+### Building function code for Graviton2
+For compiled languages like Rust and Go, you can use the `provided.al2` [custom runtime](https://docs.aws.amazon.com/lambda/latest/dg/runtimes-custom.html), which supports Arm. You provide a binary that communicates with the [Lambda Runtime API](https://docs.aws.amazon.com/lambda/latest/dg/runtimes-api.html).
+
+When compiling for Go, set `GOARCH` to `arm`.
```
Comment (AGSaidi) [inline aws-lambda/README.md:26]: arm64?

File: aws-lambda/README.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+
+## DEMO: Building Lambda functions for Graviton2
+Demo requirements: 
+* [AWS Serverless Application Model (AWS SAM)](https://aws.amazon.com/serverless/sam/)
+* [Docker](https://docs.docker.com/get-docker/)
+
+Clone the repository and change into the demo directory
+```
+git clone https://github.com/aws/aws-graviton-getting-started
+cd aws-lambda/GravitonLambdaNumber
```
Comment (AGSaidi) [inline aws-lambda/README.md:58]: `cd aws-graviton-getting-started` too?

File: aws-lambda/README.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+The Lambda function should respond with the `arm64` processor architecture and a fact about the date.
+
+![curl arm64](/aws-lambda/img/curlarm64.png)
+
+The function has seamlessly migrated from x86 to arm64.
+
+### Building a Lambda function with binary dependencies
+
+This function has no binary dependencies. If you do have a function that required compilation for arm64, AWS SAM can use an arm64 build container image to create the artifacts for arm64.
+This allows you to build arm64 specific dependencies on an x86 workstation.
```
Comment (AGSaidi) [inline aws-lambda/README.md:122]: does it work on an M1 mac too?

**Review by AGSaidi — COMMENTED** [review]
Thanks for this. It's awesome!

File: aws-lambda/README.md (hunk 1 above)
Comment (julianwood) [reply aws-lambda/README.md]: Amended in new edit

File: aws-lambda/README.md (hunk 2 above)
Comment (julianwood) [reply aws-lambda/README.md]: Amended in new edit

File: aws-lambda/README.md (hunk 1 above)
Comment (julianwood) [reply aws-lambda/README.md]: Amended in new edit

File: aws-lambda/README.md (hunk 3 above)
Comment (julianwood) [reply aws-lambda/README.md]: Amended wording to specify it works in either direction.

**Comment by julianwood** [comment]
Amended wording based on feedback

**Pull request merged**